### PR TITLE
Adamjones/aks kubernetes

### PR DESCRIPTION
**Pull request by adds68**

### CODE REVIEW

File: azure/aks-kubernetes.md [hunk 1]
```diff
@@ -30,9 +30,14 @@ The first is at the Azure service level.
...
+| Azure Kubernetes Service RBAC Writer   | Allows read/write access to most objects in a namespace. This role doesn't allow viewing or modifying roles or role bindings. However, this role allows accessing Secrets and running Pods as any ServiceAccount in the namespace, so it can be used to gain the API access levels of any ServiceAccount in the namespace. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+| Azure Kubernetes Service RBAC Admin   | Allows admin access, intended to be granted within a namespace. Allows read/write access to most resources in a namespace (or cluster scope), including the ability to create roles and role bindings within the namespace. This role doesn't allow write access to resource quota or to the namespace itself. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+| Azure Kubernetes Service RBAC Cluster Admin   |Allows super-user access to perform any action on any resource. It gives full control over every resource in the cluster and in all namespaces. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+||||
```
Comment (TLATER) [inline azure/aks-kubernetes.md:39]: This adds an empty row at the end of the table ;)

File: azure/aks-kubernetes.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This can be done by AKS, but it is recommended to use the integration with Azure
 Using AD lets you manage users in a central place that can be used across many different Azure services.
 
 1.  https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/
-2.  https://docs.microsoft.com/en-us/azure/aks/concepts-identity
+2.  https://docs.microsoft.com/en-gb/azure/aks/security-baseline#31-maintain-an-inventory-of-administrative-accounts
```
Comment (TLATER) [inline azure/aks-kubernetes.md:12]: I think the `en-us` docs are probably the canonical ones, given that you're forwarded to them if you strip the language tag.

**Review by TLATER — CHANGES_REQUESTED** [review]

File: azure/aks-kubernetes.md (hunk 2 above)
Comment (adds68) [reply azure/aks-kubernetes.md:12]: Sure, i feel like the is a little more relevant, but now the table is clearer i am not too fussed

File: azure/aks-kubernetes.md (hunk 1 above)
Comment (adds68) [reply azure/aks-kubernetes.md:39]: You're right, but it does look pretty....

**Comment by TLATER** [comment]
Hmm, I didn't set the default branch correctly so GitHub didn't pick up my rebase. Merged in 475f471b000e7a3ef3dee0b016e0d70da560aa03.

**Pull request closed without merging**